plugins/modules/zos_operator_action_query.py

```diff
@@ -395,35 +396,35 @@ def filter_requests(merged_list, params):
     message_id = params.get("message_id")
     job_name = params.get("job_name")
     newlist = merged_list
-
     if system:
         newlist = handle_conditions(newlist, "system", system)
     if job_name:
         newlist = handle_conditions(newlist, "job_name", job_name)
     if message_id:
         newlist = handle_conditions(newlist, "message_id", message_id)
-
     return newlist
 
 
-def handle_conditions(list, condition_type, value):
+def handle_conditions(merged_list, condition_type, value):
     # regex = re.compile(condition_values)
     newlist = []
-    for dict in list:
-        if value.endswith("*"):
-            exist = dict.get(condition_type).startswith(value.rstrip("*"))
-        else:
-            exist = dict.get(condition_type) == value
+    exist = False
+    for message in merged_list:
+        if message.get(condition_type) is not None:
+            if value.endswith("*"):
+                exist = message.get(condition_type).startswith(value.rstrip("*"))
+            else:
+                exist = message.get(condition_type) == value
 
         if exist:
-            newlist.append(dict)
+            newlist.append(message)
     return newlist
 
 
-def execute_command(operator_cmd, timeout=1, *args, **kwargs):
-
-    # response = opercmd.execute(operator_cmd)
-    response = opercmd.execute(operator_cmd, timeout, *args, **kwargs)
+def execute_command(operator_cmd, timeout_s=1, *args, **kwargs):
```

Comment:
remember to update lines 276 and 291 where this function is called: `timeout` --> `timeout_s`

`cmd_result_a = execute_command(cmdtxt, timeout=wait_s, *args, **kwargs)`
`cmd_result_b = execute_command(cmdtxt, timeout=wait_s, *args, **kwargs)`